scripts/utilities/003_update_project_field_value_couchdb_2_x.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+# -----------------------------------------------------------------------------
+# Copyright Siemens AG, 2019. Part of the SW360 Portal Project.
```

Comment:
May be the Copyright year should be updated.

Reply:
done